model-view-intent/src/main/java/com/iluwatar/model/view/intent/CalculatorView.java

```diff
@@ -0,0 +1,70 @@
+package com.iluwatar.model.view.intent;
+
+import com.iluwatar.model.view.intent.actions.AdditionCalculatorAction;
+import com.iluwatar.model.view.intent.actions.DivisionCalculatorAction;
+import com.iluwatar.model.view.intent.actions.MultiplicationCalculatorAction;
+import com.iluwatar.model.view.intent.actions.SetVariableCalculatorAction;
+import com.iluwatar.model.view.intent.actions.SubtractionCalculatorAction;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Exposes changes to the state of calculator
+ * to {@link CalculatorViewModel} through
+ * {@link com.iluwatar.model.view.intent.actions.CalculatorAction}
+ * and displays its updated {@link CalculatorModel}.
+ */
+@Slf4j
+public class CalculatorView {
+
+  /**
+   * View model param handling the operations.
+   * */
+  private final CalculatorViewModel viewModel = new CalculatorViewModel();
```

Comment:
Usually it's better to pass variables through constructors or methods to avoid tight coupling to single implementation